### PR TITLE
util: refactor logs.

### PR DESCRIPTION
**Pull request by hicqu**
Signed-off-by: qupeng <qupeng@pingcap.com>

## What have you changed? (mandatory)
Refactor all logs in util module.

## What are the type of the changes? (mandatory)
Improvement, for code style.

## How has this PR been tested? (mandatory)
CI.

## Does this PR affect documentation (docs) update? (mandatory)
NO.

## Does this PR affect tidb-ansible update? (mandatory)
No.

### CODE REVIEW

File: src/util/worker/mod.rs [hunk 1]
```diff
@@ -340,9 +340,9 @@ impl<T: Display + Send + 'static> Worker<T> {
         U: Send + 'static,
     {
         let mut receiver = self.receiver.lock().unwrap();
-        info!("starting working thread: {}", self.scheduler.name);
+        info!("starting working thread"; "worker" => &self.scheduler.name);
```
Comment (AndreMouche) [inline src/util/worker/mod.rs]: could we remove `&` here?

File: src/util/macros.rs [hunk 2]
```diff
@@ -105,7 +105,10 @@ macro_rules! box_err {
 macro_rules! slow_log {
     ($t:expr, $($arg:tt)*) => {{
         if $t.is_slow() {
-            warn!("{} [takes {:?}]", format_args!($($arg)*), $t.elapsed());
+            warn!("handle task slow";
+                "task" => format!("{}", format_args!($($arg)*)),
+                "elapsed" => ?$t.elapsed(),
```
Comment (overvenus) [inline src/util/macros.rs]: How about `takes`?

File: src/util/macros.rs [hunk 3]
```diff
@@ -105,7 +105,10 @@ macro_rules! box_err {
 macro_rules! slow_log {
     ($t:expr, $($arg:tt)*) => {{
         if $t.is_slow() {
-            warn!("{} [takes {:?}]", format_args!($($arg)*), $t.elapsed());
+            warn!("handle task slow";
+                "task" => format!("{}", format_args!($($arg)*)),
```
Comment (overvenus) [inline src/util/macros.rs]: What about `warn!($($arg)*, "takes" => ?$t.elapsed())`?

File: src/util/worker/mod.rs (hunk 1 above)
Comment (hicqu) [reply src/util/worker/mod.rs]: Currently we can't.

File: src/util/macros.rs [hunk 4]
```diff
@@ -105,10 +105,7 @@ macro_rules! box_err {
 macro_rules! slow_log {
     ($t:expr, $($arg:tt)*) => {{
         if $t.is_slow() {
-            warn!("handle task slow";
-                "task" => format!("{}", format_args!($($arg)*)),
-                "elapsed" => ?$t.elapsed(),
-            );
+            warn!($($arg)*; "takes" => ?$t.elapsed());
```
Comment (overvenus) [inline src/util/macros.rs]: What if we want log more `kv`s?

Comment (hicqu) [reply src/util/macros.rs]: So we can extend the macro when we need this.

**Pull request merged**